### DISCUSSION TITLE
How make transition for item QVirtualScroll

### DISCUSSION THREAD

**Discussion by Gumper-x** [quasarframework/quasar#13199] [General - Components / Directives / etc]
I made this:
```vue
<q-virtual-scroll
  :items="Array(50).fill('test')"
  virtual-scroll-horizontal
>
    <template v-slot="{ item }">
        <transition name="scale" appear>
            <q-chip
              color="blue-8"
              text-color="white"
              icon="bookmark_border"
            >
              {{ item }}
            </q-chip>
        </transition>
    </template>
</q-virtual-scroll>
```
But the problem is that this only works for appearing and adding to the array.
When we remove an element, it is removed without animation.

Help me figure out how to solve this?

**Comment by ontwikkelfabriek** [comment]
@Gumper-x 

> But the problem is that this only works for appearing and adding to the array.

I think it is because of the `appear`  inside the transition

<transition name="scale" **appear**>

**Reply by Gumper-x** [reply]
i can remove appear, but that doesn't solve the problem Normally I would use tarnation-group and v-for, and in case virtual-list so easy i can't do it

**Reply by ontwikkelfabriek** [reply]
@Gumper-x 

I have had only but problems with Quasars's virtual and Infinite scrollers. Instead I use this Vue scroller:
https://github.com/Akryum/vue-virtual-scroller
